setup.py

```diff
@@ -16,5 +16,6 @@
     install_requires=[
         'demands >= 1.0.5, < 2.0.0',
         'property-caching >= 1.0.0, < 2.0.0',
+        'six >= 1.4.0, < 2.0.0',
```

Comment:
Looks like it breaks the bundler, not us.

Because it's the earliest version bundling urllib. This is a library, broad dependencies are helpful for downstream users.